docs/probes.md

```diff
@@ -26,105 +26,114 @@ Make sure to provide enough delay and failureThreshold for the VM and the agent
-    vm.kubevirt.io/name: probe-test
-    kubevirt.io/domain: probe-test
+    kubevirt.io/vm: readiness-probe-vm
+  name: readiness-probe
 spec:
   running: false
   template:
     metadata:
       labels:
-        vm.kubevirt.io/name: probe-test
-        kubevirt.io/domain: probe-test
+        kubevirt.io/domain: readiness-probe
+        kubevirt.io/vm: readiness-probe
     spec:
-      readinessProbe:
-        exec:
-          command:
-          - cat
-          - /tmp/ready.txt
-        failureThreshold: 10
-        initialDelaySeconds: 120
-        periodSeconds: 10
-        # Note that timeoutSeconds value does not have any impact before K8s v1.20.
-        timeoutSeconds: 5
       domain:
         cpu:
           cores: 1
           sockets: 1
           threads: 1
         devices:
           disks:
-            - disk:
+            - name: containerdisk
+              disk:
                 bus: virtio
-              name: cloudinitdisk
-            - bootOrder: 1
+            - name: cloudinitdisk
               disk:
                 bus: virtio
-              name: rootdisk
           interfaces:
-            - masquerade: {}
-              model: virtio
-              name: nic-0
-          networkInterfaceMultiqueue: true
+            - name: default
+              masquerade: {}
           rng: {}
         machine:
           type: pc-q35-rhel8.2.0
         resources:
           requests:
             memory: 1Gi
-      hostname: probe-test
       networks:
-        - name: nic-0
+        - name: default
           pod: {}
+      readinessProbe:
+        exec:
+          command: ["cat", "/tmp/healthy.txt"]
+        failureThreshold: 10
+        initialDelaySeconds: 120
+        periodSeconds: 10
+        # Note that timeoutSeconds value does not have any impact before K8s v1.20.
+        timeoutSeconds: 5
       terminationGracePeriodSeconds: 180
       volumes:
+        - containerDisk:
+            image: quay.io/containerdisks/fedora
+          name: containerdisk
         - cloudInitNoCloud:
-            userData: |
+            userData: |-
               #cloud-config
-              user: fedora
-              password: fedora
               chpasswd:
                 expire: false
+              password: password
+              user: fedora
               packages:
                 qemu-guest-agent
               runcmd:
-                - [ "systemctl", "enable", "--now", "qemu-guest-agent" ]
-                - [ "touch" "/tmp/ready.txt" ]
+                - ["touch", "/tmp/healthy.txt"]
+                - ["sudo", "chcon", "--type", "virt_qemu_ga_exec_t", "/tmp/healthy.txt"]
+                - ["sudo", "systemctl", "enable", "--now", "qemu-guest-agent"]
           name: cloudinitdisk
-        - containerDisk:
-            image: kubevirt/fedora-cloud-container-disk-demo
-          name: rootdisk
 ```
 2. Apply the VM manifest
 
 ```sh
-kubectl apply -f /tmp/probe-test.yaml
+kubectl apply --filename /tmp/probe-test.yaml
```

Comment:
I'm not sure if this is worth changing tbh, if anything you can just merge steps 1 and 2 into a single apply step.

Reply:
In my humble opinion, I believe it is more educational to have all steps. Please, feel free to disagree, it's just my opinion.

Reply:
Yeah understood and not something I'd block on. IMHO however making examples as minimal as possible helps readers focus on the relevant content of the example, in this case the `readinessProbe` and not the lifecycle of a VM.

Reply:
Ok, I agree. I will merge steps 1 and 2. Thanks for the inputs!